etcdserver/etcdserverpb/raft_internal.proto

```diff
@@ -35,6 +35,7 @@ message InternalRaftRequest {
   AuthenticateRequest authenticate = 17;
 
   AlarmRequest alarm = 18;
+  AuthDisableRequest auth_disable = 19;
```

Comment:
I have added it to the end. Ideally it would be better to have it next to AuthEnable. Since v3 is not released yet shall I move it next to AuthEnable?